README.md

```diff
@@ -1,8 +1,8 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# WEBPACK BOILERPLATE
+# Leaderboard
 
-> This is a template to start a JS project using webpack.
+> This is an app where you can check and submits scores usings APIs.
```

Comment:
- [ ] Kindly deploy the project online and include the live link in the readme.

Reply:
Thanks, Sir. Done.